pfs/camera/camera.py

```diff
@@ -19,7 +19,7 @@
 config.detectorList={}
 config.detectorList[0]=lsst.afw.cameraGeom.cameraConfig.DetectorConfig()
 config.detectorList[0].bbox_y0=0
-config.detectorList[0].bbox_y1=4173
+config.detectorList[0].bbox_y1=4175
```

Comment:
NROW = 4172  # CCDROWS(=4224) - LEADINROWS(=48) (=4176)
means that there is a mistake in the datamodel